### ISSUE TITLE
Images are busted - only blurred versions are shown

### ISSUE THREAD

**Issue by Servelan** [elk-zone/elk#2305]
This is brand new - yesterday, images showed up just fine.  Today I noticed it beginning with the image of someone's craftwork being blurry; clicked on the image and the alt text was displayed with what appeared to be 'blank' space where the image should be.   

Link to post:  https://elk.zone/social.afront.org/@terri/110832697739797924

**Comment by Servelan** [comment]
This is caused by media/object storage issues with Vultr; not a bug.  Newsie.social says Vultr is working on it.
